packages/docs-theme/src/styles/_examples.scss

```diff
@@ -1,32 +1,34 @@
 // Copyright 2017 Palantir Technologies, Inc. All rights reserved.
 // Licensed under the terms of the LICENSE file distributed with this project.
 
-// Base Styles for All Examples
+// Base styles for all examples
 
-// TODO: dark support
+$example-frame-spacing: $pt-grid-size;
+$example-frame-border-radius: $pt-border-radius * 2;
+$example-spacing: $pt-grid-size * 4;
+
+$example-background-color: $white;
+$code-background-color: $light-gray4;
+$options-background-color: $light-gray4;
+
+$dark-example-background-color: $dark-gray2;
+$dark-code-background-color: $dark-gray4;
+$dark-options-background-color: $dark-gray4;
```

Comment:
BLUF of this file changes is variables for easier tweaking, and a less harsh example frame--they're all independently rounded so they work well with or without options/markup